src/main/java/sirius/biz/tenants/TenantUserManager.java

```diff
@@ -493,7 +493,7 @@ public UserInfo findUserByCredentials(@Nullable WebContext ctx, String user, Str
         }
 
         LoginData loginData = account.getUserAccountData().getLogin();
-        if (acceptApiTokens && Strings.areEqual(password, loginData.getApiToken())) {
+        if (acceptApiTokens && loginData.checkApiToken(password)) {
```

Comment:
shouldn't we OR the latter?

Reply:
i don't think so, `acceptApiTokens` is a configuration, if apiTokens should be possible at all, so an AND seems correct to me